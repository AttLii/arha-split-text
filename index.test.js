import React from "react";
import renderer from "react-test-renderer";
import SplitText from "./index";

describe("<SplitText>", () => {
  it("Matches the snapshot", () => {
    const splitText = renderer.create(<SplitText>Test</SplitText>);
    expect(splitText.toJSON()).toMatchSnapshot();
  });
});
