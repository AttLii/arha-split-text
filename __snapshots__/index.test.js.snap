// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SplitText> Matches the snapshot 1`] = `
<div
  aria-label="Test"
  className="c-split-text "
>
  <span
    aria-hidden={true}
    className="word "
    style={
      Object {
        "display": "inline-block",
      }
    }
  >
    <span
      aria-hidden={true}
      className="char "
    >
      T
    </span>
    <span
      aria-hidden={true}
      className="char "
    >
      e
    </span>
    <span
      aria-hidden={true}
      className="char "
    >
      s
    </span>
    <span
      aria-hidden={true}
      className="char "
    >
      t
    </span>
  </span>
</div>
`;
